scene/main/viewport.cpp

```diff
@@ -4752,7 +4776,8 @@ void Viewport::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_vrs_texture"), &Viewport::get_vrs_texture);
 
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "disable_3d"), "set_disable_3d", "is_3d_disabled");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_xr"), "set_use_xr", "is_using_xr");
+	//ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_xr"), "set_use_xr", "is_using_xr");
```

Comment:
I think we need to keep `use_xr` as a property but marked as deprecated for a few versions. We're going to be dealing with a lot of toolkits that need to work both with current and with older versions of Godot that use this property to enable XR.

Reply:
I'm fine with just leaving it exposed for backwards compatibility. I commented it out just for testing, I will uncomment it before removing this PR from draft (if we decide to move forward with these changes)